### ISSUE TITLE
As a temp outfitter and guide, I should see an estimate of how much my permit will cost, so  that I can have more predictable costs.

### ISSUE THREAD

**Issue by lauraGgit** [USDAForestService/fs-open-forest-platform#348]
## Notes
Right now we are collecting the inputs to calculate an estimated cost of the permit, but do not provide any estimate outside of the [Learn more page](https://forest-service-trees-staging.app.cloud.gov/applications/temp-outfitters/learn-more)

Update: Story is essential for a positive user experience. People typically want an idea of how much something might cost them.

[Special Uses Fee Determination FSH.pdf](https://github.com/USDAForestService/fs-open-forest-platform/files/3768606/Special.Uses.Fee.Determination.FSH.pdf) see section 37.21b - Flat Fee for Temporary Use Permits on page 68 of the document.

## Acceptance Criteria
- [x] A temp OG is informed of how much their Special Use permit may cost.
- [x] A temp OG is informed that the final determination of how much their permit will cost will be decided by the Special Use permit administrator after their application is reviewed.

## Tasks
- [x] Designer, researcher and PO to meet and discuss the best solution.
- [x] Mock up the following:
- [x] **In Step 4 Client Charges, revise the description to: "Provide description of the fees you plan to charge your clients, and what those fees would cover."**
- [x] **Under Step 4: Client Charges. Edit the first sentence of hint text to read: "Your description of client fees will not impact what you would be charged for this permit. Charges for a temporary outfitting and guiding permit are based on a flat rate."**
- [x] **Insert a header before Step 3: Advertising section Titled:  "Estimate your permit cost"  (this is not a new "Step", just a new header)**
- [x] **Insert the Flat Rate information and Cost Chart in this "Estimate your permit cost" section.**
- [x] **Learn More page, remove: "Forest service handbook info"(See FSH 2709.11, sec. 37.21c)"**  because this reference is for liability insurance estimates, not permit cost estimates.
- [x] PO and Design Team meets to review/refine
- [x] Mock up finalized. tasks (or a new story) for Dev are created.
- [x] PO approved.

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] Server actions captured by logs
- [x] No new security vulnerabilities
- [x] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [x] Compare finished design with mockup
- [ ] Usability tested

**Comment by ASprinkle** [comment]
Temp OG permits are good for 180 days and limited to 200 User Days. A user day is defined as the number of people in a single day using a day (i.e., 1 person on one day = 1 user day.  10 people on one day =10 user days. 10 People on a 10-day trip = 100 user days. 10 people in the morning and 10 people in the afternoon on the same day = 20 user days. 100 trips and 50 people per trip would be denied, not enough user days.).
For reference FSH 2709.11, CH 30, Section 37.21(b)

Number of User Days | Flat Fee | 
 1 to 50                            | $150 |
 51 to 100                         | $300 | 
101 to 150                        | $450 | 
151 to 200                        | $600 |

**Comment by lisamcmichael** [comment]
There is information in the Learn More page. However, seems like there should be a calculator/estimator. But for MVP, we'll use what we have.  

Maybe add a learn more link to see what are the flat fees?!

**Comment by carlsonem** [comment]
@lisamcmichael @lisaredux Please update the LOE if you feel that is off after taking a look at this issue

**Comment by lisamcmichael** [comment]
New paragraph verbiage: Please provide description of  client charges and fees. 

Learn more page: Remove forest service handbook info"(See FSH 2709.11, sec. 37.21c)"

**Comment by lisamcmichael** [comment]
@ASprinkle 2 weeks ago we decided to make these updates. Would it be jumping the gun to move it to In Progress, and have Lisa S. mockup the changes when she gets back?


For the first paragraph, revise it to: "Please provide description of client charges and fees."
Also, on the Learn More page, remove: "Forest service handbook info"(See FSH 2709.11, sec. 37.21c)"

**Comment by ASprinkle** [comment]
I think a mock up will be a great plan. there is more to it than just those changes... I'll work on updating the A/C

**Comment by lisaredux** [comment]
Tackling this one after story 530

**Comment by lisamcmichael** [comment]
Met in Dev/Design today, and decided that @lisaredux will setup environment and use this story as a test case. She is working with @briandavidson and plans to have an update by Monday/Tuesday.

**Comment by lisaredux** [comment]
Working on mockups today. Implementing without setting up my local enviroment first

**Comment by lisaredux** [comment]
@ASprinkle @lisamcmichael Mockups are ready for your review:

**APPLICATION PAGE**
![Screenshot-ApplicationPage-348](https://user-images.githubusercontent.com/26801590/70003385-f0c0c480-1530-11ea-952a-4a68b2c68612.png)

**LEARN MORE PAGE**
<img width="1234" alt="Screenshot 2019-12-02 18 05 06" src="https://user-images.githubusercontent.com/26801590/70003425-1352dd80-1531-11ea-9b90-e87045e65622.png">

**Comment by ASprinkle** [comment]
@lisaredux  These look great. I notices one typo that I made in the tasks:

The bold font in Client Charges: " Provide a description..." we need the word/letter "a" there.

Also, i noticed that we do not want to create an expectation that they will be charged one of the fees... we want to emphasize that it can be used as an estimate. I think we do that well in the application page given it says "Estimate your permit cost".

However, in the learn more page we call it "cost chart" We should name it: "Cost estimate chart" and the header should be "Cost estimate" to be consistent with the application page.

**Comment by lisaredux** [comment]
@ASprinkle here are the updated designs

@briandavidson @mtlaney ready for DEV

**TEMP OUTFITTERS APPLICATION PAGE**

![Story348-TempApply](https://user-images.githubusercontent.com/26801590/70071622-9fade080-15c3-11ea-957a-255c506c0765.png)

**TEMP OUTFITTERS LEARN MORE PAGE**

<img width="1169" alt="Screenshot 2019-12-03 11 57 06" src="https://user-images.githubusercontent.com/26801590/70071921-08955880-15c4-11ea-9c41-fc6f61c75185.png">

**Comment by ASprinkle** [comment]
@briandavidson 

The chart and estimate section in step 3 looks great!
![image](https://user-images.githubusercontent.com/33435621/71585961-44411680-2acd-11ea-8765-9a83034467a7.png)

It looks like I am still seeing the link for Forest service handbook  FSH 2709.11, sec. 37.21c) in staging. Can you also remove the entire line of text and link "(See FSH 2709.11, sec 37.12c)" as part of the tasks above?  Here is a screen shot with the line of text circled:
![image](https://user-images.githubusercontent.com/33435621/71585987-5753e680-2acd-11ea-8822-bc87adb3d596.png)

Also, what I am seeing in staging -it does not look like the text under section "step 4" has been updated.
![image](https://user-images.githubusercontent.com/33435621/71586001-676bc600-2acd-11ea-94e1-a052ea4ac47d.png)

**Comment by ASprinkle** [comment]
@briandavidson can you remove the last sentence, shown here as the red line strike through?
![image](https://user-images.githubusercontent.com/33435621/71634926-c5cf9c00-2bd4-11ea-8806-20c2de499e69.png)

**Comment by ASprinkle** [comment]
![image](https://user-images.githubusercontent.com/33435621/72178917-d16f4f80-3398-11ea-81bb-40b9d641d688.png)
